src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4385 @@
+---
+id: scripting_advanced_race_timer
+name: Creating a Race Timer in Core (Advanced)
+title: Creating a Race Timer in Core (Advanced)
+tags:
+    - Tutorial
+---
+
+# Advanced Scripting in Core
+
+## Overview
+
+In this tutorial we are going to explore more of the **Core** API to make a race timer for a multiplayer game from start to finish.
+
+* **Completion Time:** ~3 hours
+* **Knowledge Level:** It's recommended to have completed the [Scripting Beginner](lua_basics_helloworld.md) and [Scripting Intermediate](lua_basics_lightbulb.md) tutorials.
+* **Skills you will learn:**
+    * When to use Client and Server contexts
+    * Accurate time keeping
+    * Broadcasting to the client
+    * Persistent storage
+    * Sending and receiving private data
+    * Submitting and retrieving leaderboard data
+
+---
+
+## Setting up Project and Track
+
+Start with making a new empty project.
+
+![!New Empty Project](../img/RaceTimerTutorial/create_new_project.png){: .center loading="lazy" }
+
+In the **Hierarchy** delete the object **Default Floor** as we will not be needing this for the track.
+
+![!Remove Default Floor](../img/RaceTimerTutorial/remove_default_floor.png){: .center loading="lazy" }
+
+We now need to create a track for players to run on. A track has been uploaded to **Community Content** to help you get started. This can be imported from the **Community Content** panel by searching for **Race Timer Tutorial - Track** by **CommanderFoo**.
+
+![!Community Content Template](../img/RaceTimerTutorial/import_track.png){: .center loading="lazy" }
+
+Drag and drop the template **Race Timer Tutorial - Track** into the **Hierarchy** and **Deinstance** it.
```

Comment:
right click and select **Deinstance This Template**

Essentially, we need to decide if the readership is starting here (possible, if they have some experience elsewhere but want to learn Core) or if we're going to assume they've done some other tutorials. If the former, we should explain the mechanics of deleting and deinstancing explicitly, and if the latter, we probably don't need the images.

I'm not sure which is correct here, but I'm leaning towards the second scenario. Most of the demand for this is from creators who have already done Lightbulb, and advanced creators have the option of looking back if they can't figure out a step.

Reply:
I agree, the second scenario is better, since this is aimed at more experienced scripters. 

I've cut out some of the beginner type stuff in the early section, and removed a few images and videos.